domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/datamapper/domene/StønadskontoMapper.java

```diff
@@ -17,19 +17,29 @@
 import no.nav.foreldrepenger.melding.uttak.StønadskontoType;
 import no.nav.foreldrepenger.melding.uttak.UttakResultatPerioder;
 
-public class StønadskontoMapper {
+public final class StønadskontoMapper {
```

Comment:
Finner ingen tester for denne klassen. Vurder å skrive noen tester for hver av public metodene.